cmake/ctest_script.cmake

```diff
@@ -55,53 +72,55 @@ set(CTEST_CMAKE_GENERATOR "Unix Makefiles")
 #set(ENV{CC}  "clang")
 #set(ENV{CXX} "clang++")
 
-# Build type
-set(CTEST_BUILD_CONFIGURATION RelWithDebInfo)
-
-# Name of this build
-find_program(UNAME NAMES uname)
-exec_program("${UNAME}" ARGS "-s" OUTPUT_VARIABLE osname)
-exec_program("${UNAME}" ARGS "-m" OUTPUT_VARIABLE "cpu")
-set(CTEST_BUILD_NAME "${osname}-${cpu} ${CTEST_CMAKE_GENERATOR}/${CTEST_BUILD_CONFIGURATION} $ENV{CXX}")
-
-# Test type (Nightly, Continuous, Experimental)
-set(test_model "Experimental")
+# Build options
+set(gismo_build_options 
+    -DGISMO_BUILD_LIB=ON
+    -DGISMO_WITH_ONURBS=ON
+    #-DGISMO_WITH_IPOPT=ON
+    #-DIpOpt_DIR=/path/to/ipopt
+    #-DGISMO_WITH_PSOLID=ON
+    #-DParasolid_DIR=/path/to/parasolid
+    #-DGISMO_BUILD_AXL=ON -DAxel_DIR=/path/to/axel
+    #-DGISMO_PLAINDOX=ON
+    #-DGISMO_BUILD_COVERAGE=ON
+    -DGISMO_BUILD_UNITTESTS=ON
+)
 
 # For continuous builds, number of seconds to stay alive
 set(test_runtime 40000)
 
-# Build flags
-set(CTEST_BUILD_FLAGS "-j2")
+# ID for this computer that shows up on the dashboard.
+if(NOT HOSTNAME)
+find_program(HOSTNAME_CMD NAMES hostname)
+exec_program(${HOSTNAME_CMD} ARGS OUTPUT_VARIABLE HOSTNAME)
+set(CTEST_SITE "${HOSTNAME}")
+endif(NOT HOSTNAME)
 
-# Source folder
-set(CTEST_SOURCE_DIRECTORY ${CTEST_SCRIPT_DIRECTORY}/..)
+# Name of this build
+if(NOT CTEST_BUILD_NAME)
+find_program(UNAME NAMES uname)
+exec_program("${UNAME}" ARGS "-s" OUTPUT_VARIABLE osname)
```

Comment:
deprecated, execute_process should be used instead:
https://cmake.org/cmake/help/v3.8/command/exec_program.html